Model/Generator.php

```diff
@@ -520,19 +521,21 @@ public function getCheckoutSession()
     {
         return $this->checkoutSession;
     }
+
     /**
-     * @param $order
+     * @param order $order
```

Comment:
`order` doesn't appear to be a class, was it supposed to be `Order`?

Reply:
Done